engine/src/main/java/org/terasology/world/block/Block.java

```diff
@@ -587,27 +584,6 @@ public AABBf getBounds(Vector3f floatPos) {
         return getBounds(new Vector3i(floatPos, RoundingMode.HALF_UP));
     }
 
-    public void renderWithLightValue(float sunlight, float blockLight) {
-        if (meshGenerator == null) {
-            return;
-        }
-
-        Material mat = Assets.getMaterial("engine:prog.block").orElseThrow(() -> new RuntimeException("Missing engine material"));
```

Comment:
Is this material specific to this method, or is it used in other places, too?

Reply:
I think this is the only place it seems?